pages/api/posts/publish.js

```diff
@@ -1,16 +1,21 @@
-import auth0 from "../../src/lib/auth0";
+import auth0 from "../../../src/lib/auth0";
 import GitHubPublisher from "github-publish";
-import { sanitize } from "../../src/lib/teams";
-import { isAllowedToPost } from "../../src/isAllowedToPost";
+import { isAllowedToPost } from "../../../src/isAllowedToPost";
 
-const token = process.env.GH_TOKEN;
 const org = process.env.GH_ORG;
 const repo = process.env.GH_REPO;
+const token = process.env.GH_TOKEN;
 const branch = process.env.GH_BRANCH;
 const path = process.env.GH_PATH || "content";
 
 const pad = num => (parseInt(num, 10) < 10 ? "0" + parseInt(num, 10) : num);
 
+const sanitize = name =>
+  name
+    .replace(/[éè]/g, "e")
+    .replace(/[\s/;:,!?#]/g, "-")
+    .toLowerCase();
+
```

Comment:
seen that somewhere before ? :)

in `publishTeam` and `publishLatest` the `...args` and `args[0]` sounds over complicated

one could use smthing more explicit like `const publishTeam = ({ author, data }) => {...}`

Reply:
If we speak about `sanitize` it's now present only in this file and used only to process publisher name. For the `teams`, slugs are now provided by Github api.